main.go

```diff
@@ -90,6 +103,15 @@ It can be used in a curl-like fashion when benchmarking features are disabled.
 		return opts, errExit
 	}
 
+	// Read defaults from ~/.yab.ini
+	if user, err := user.Current(); err != nil {
+		out.Fatalf("Failed to find the current user: %v\n", err)
+	} else {
+		homeDir := user.HomeDir
+		iniParser := flags.NewIniParser(parser)
+		iniParser.ParseFile(path.Join(homeDir, ".yab.ini"))
```

Comment:
`ParseFile` can return an error, we might want to return an error to the user in that case